packages/office-addin-manifest/src/manifestInfo.ts

```diff
@@ -42,30 +42,84 @@ function parseManifest(xml: Xml): ManifestInfo {
   const officeApp: Xml = xml.OfficeApp;
 
   if (officeApp) {
-    const defaultSettingsXml: Xml = xmlMethods.getXmlElement(officeApp, "DefaultSettings");
+    const defaultSettingsXml: Xml = xmlMethods.getXmlElement(
+      officeApp,
+      "DefaultSettings"
+    );
 
     manifest.id = xmlMethods.getXmlElementValue(officeApp, "Id");
-    manifest.allowSnapshot = xmlMethods.getXmlElementValue(officeApp, "AllowSnapshot");
-    manifest.alternateId = xmlMethods.getXmlElementValue(officeApp, "AlternateId");
-    manifest.appDomains = xmlMethods.getXmlElementsValue(officeApp, "AppDomains", "AppDomain");
-    manifest.defaultLocale = xmlMethods.getXmlElementValue(officeApp, "DefaultLocale");
-    manifest.description = xmlMethods.getXmlElementAttributeValue(officeApp, "Description");
-    manifest.displayName = xmlMethods.getXmlElementAttributeValue(officeApp, "DisplayName");
-    manifest.highResolutionIconUrl = xmlMethods.getXmlElementAttributeValue(officeApp, "HighResolutionIconUrl");
-    manifest.hosts = xmlMethods.getXmlElementsAttributeValue(officeApp, "Hosts", "Host", "Name");
-    manifest.iconUrl = xmlMethods.getXmlElementAttributeValue(officeApp, "IconUrl");
-    manifest.officeAppType = xmlMethods.getXmlAttributeValue(officeApp, "xsi:type");
-    manifest.permissions = xmlMethods.getXmlElementValue(officeApp, "Permissions");
-    manifest.providerName = xmlMethods.getXmlElementValue(officeApp, "ProviderName");
-    manifest.supportUrl = xmlMethods.getXmlElementAttributeValue(officeApp, "SupportUrl");
+    manifest.allowSnapshot = xmlMethods.getXmlElementValue(
```

Comment:
Not really prettier. Is there a line length of 120 being used?

Reply:
I'm using office-addin-lint . . . so what ever settings existed are the ones that were used.

Reply:
looks like our own prettier config package has established the 120 with setting (only thing it does).

Reply:
Looks like I needed to add a reference to office-adding-prettier-config for it to take effect and us 120 as the width.  Will look into using that everywhere.